### DISCUSSION TITLE
Listening to Mouse Wheel

### DISCUSSION THREAD

**Discussion by woosanghun** [visgl/deck.gl#6931] [Q&A] [answered]
One of the issues we have is when we put a map on a dashboard and scroll, when the mouse goes over the map it starts zooming.  So we turned it off, but we'd like to control zoom through a simple slider on the right side of the panel, so if a user hovers over the control, the zooming works with the mouse wheel. 

However, when we do this, it's as if the map prevents the component from getting the mouse wheel events.   Is there any way to have our control get those events? 

Any help on this is greatly appreciated.

**Comment by Pessimistress** [comment, marked as answer]
I assume you are using React. The `DeckGL` component watches all events under its root element. If you don't want DeckGL to see the event, just render your component outside of `DeckGL`.

**Reply by woosanghun** [reply]
Yup, that did the trick.  Thanks!
